README.md

```
@@ -1,135 +1,102 @@
+├── 📂 components        # 공통 컴포넌트
+├── 📂 lib               # 공통 함수
+└── 📂 styles            # 전역 스타일 정의
 ```
 
-### Develop
-
-To develop all apps and packages, run the following command:
-
+## 🏃실행 방법
+pnpm 및 필요 라이브러리 설치
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)